src/utils/format-date.js

```diff
@@ -51,7 +51,7 @@ function getTwelveHour(time) {
   return time
 }
 
-export function convertUnix(integer) {
+export function convertUnix(integer = 0) {
```

Comment:
Be a little more verbose on this function name, just a suggestion: `convertUnixToFormatedDate`

Reply:
I’ll make that change and fix unit test, :facepalm: